### DISCUSSION TITLE
Is there a way to edit My Account page? No block editor access, as it seems

### DISCUSSION THREAD

**Discussion by daotw** [woocommerce/woocommerce#43113] [General]
Hi,

Is there a way to edit My Account page? The standard way (through Site editor > Template, and Page) doesn't exist. All I have is [woocommerce_my_account] on the block editor (Page).

Yep, no Gutenberg access to edit the basic things like Align Wide... etc.

Right now the My Account page looks "unfinished" out of the box. There's overlapping in mobile view (without custom CSS):
<img width="405" alt="temp3" src="https://github.com/woocommerce/woocommerce/assets/31888729/a0f8fa4c-d683-48ae-bdb3-90912611affd">

I expected to see flexbox layout, but none...

**Comment by igorradovanov** [comment]
Hi there,

The `[woocommerce_my_account]` is a shortcode and can't be customized via GUI/Gutenberg, but it's quite flexible, and you can re-order, rename, and customize other parts using the `woocommerce_account_menu_items` filter.

Let's say you'd like to wrap each element in a flexbox container; you can do something like this:

```php
function my_account_menu_order($menu_items) {
    $reordered_items = array(
        'dashboard'        => __('Dashboard', 'woocommerce'),
        'downloads'        => __('Download', 'woocommerce'),
        'edit-address'     => __('Addresses', 'woocommerce'),
        'edit-account'     => __('Account Details', 'woocommerce'),
        'customer-logout'  => __('Logout', 'woocommerce'),
        'orders'           => __('Orders', 'woocommerce')
    );

    // Wrap each menu item with a flexbox container
    foreach ($reordered_items as $key => $label) {
        $reordered_items[$key] = '<div class="flex-container">' . $label . '</div>';
    }

    return $reordered_items;
}

add_filter('woocommerce_account_menu_items', 'my_account_menu_order');
